### PR TITLE
making docker work

### PR DESCRIPTION
**Pull request by SrNski**
Readme updated for how to make docker run work

should be able to reference endpoint/connect with frontend with docker run
Some properties need to be set for image run. Also in readme

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,8 +1,21 @@
 spring.application.name=demo
 spring.custom.organization.name = amplimindcc
-spring.docker.compose.lifecycle-management=start_only
 
+
+##Enable for development
 spring.datasource.url=jdbc:postgresql://localhost:5432/postgres_db
+springdoc.swagger-ui.enabled=true
+spring.docker.compose.lifecycle-management=start_only
+
```
Comment (SrNski) [inline src/main/resources/application.properties:9]: Enable for development: need to be enabled for IDE run
Enable for image build: need to be enabled for creating a working image

Only one set shall be enabled at a time

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -13,10 +26,6 @@ spring.jpa.hibernate.ddl-auto=create
 #TODO change for correct servername
 app.frontend.url=http://localhost:5173
 
-#(de)activate the swagger ui
-springdoc.swagger-ui.enabled=true
```
Comment (SrNski) [inline src/main/resources/application.properties:17]: moved

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,8 +1,21 @@
 spring.application.name=demo
 spring.custom.organization.name = amplimindcc
-spring.docker.compose.lifecycle-management=start_only
```
Comment (SrNski) [inline src/main/resources/application.properties:3]: moved

File: docker-compose.yml [hunk 4]
```diff
@@ -20,10 +20,14 @@ services:
 
   app:
     build: ./
+    depends_on:
```
Comment (SrNski) [inline docker-compose.yml:23]: wait for dependent services to run before launching app

File: Dockerfile [hunk 5]
```diff
@@ -1,9 +1,5 @@
...
-
-COPY . .
-
-RUN chmod +x gradlew
-
-RUN ./gradlew build
\ No newline at end of file
+ARG JAR_FILE=build/libs/*.jar
+COPY ${JAR_FILE} app.jar
+ENTRYPOINT ["java","-jar","/app.jar"]
```
Comment (SrNski) [inline Dockerfile:5]: This makes the entire image work

**Review by Stefan7400 — APPROVED** [review]

**Pull request merged**